01.php.md

```diff
@@ -54,15 +54,16 @@ PHP обладает, так называемой, слабой типизаци
 Использовать реализацию основанную на определении понятия "вес Хемминга".
 
 ```php
-function hammingWeight($num)
-{
+function hammingWeight($num) {
```

Comment:
Please, look at examples of function declaration http://www.php-fig.org/psr/psr-1/